process/github-policy.md

```diff
@@ -0,0 +1,82 @@
+# GitHub policies
+
+Open Mainframe Project hosted projects generally use GitHub for code hosting and issue management. GitHub organizations for hosted projects are owned and admistered by the Linux Foundation staff, including the Linux Foundation release engineering team, to ensure the sustainability of the infrastructure. 
+
+This document outlines the policies and procedures for projects using GitHub for code hosting. In addition, the Linux Foundation release engineering team maintains [documentation](https://docs.releng.linuxfoundation.org/en/latest/) on it's services, policies, and procedures.
+
+## New project or repository
+
+When a new project or repository is to be added, please [submit a request to the Open Mainframe Project staff](https://github.com/openmainframeproject/foundation/issues/new/choose) who can faciliate the process.
+
+### Code license scan
+
+If adding the new project or repository will include a code dump, it's highly recommended to request a code license scan before bringing the code in. This scan will be a validation for...
```

Comment:
Here and below, beginning with the sentence "This scan will be a validation for...", I would replace it with:

```
This scan will look for, and will provide recommendations (or in some case required prior remediation), for:

- Presence of third party licenses (OSI-approved or otherwise) which might be considered incompatible with the project's license
- Presence of headers with the project's designated license(s) and preferred copyright notices in project files (refer to the [License Specification in the Contribution Guidelines](https://github.com/openmainframeproject/tac/blob/master/process/contribution_guidelines.md#license-specification) for more information)
- Any other best practices guidance